packages/shared/src/components/post/collection/CollectionPostContent.tsx

```diff
@@ -66,6 +70,16 @@ export const CollectionPostContent = ({
     onRemovePost,
   };
 
+  const { mutateAsync: onSendViewPost } = useMutation(sendViewPost);
+
+  useEffect(() => {
```

Comment:
Since we should do it for everything except article, can we have a generic way to do it?

Reply:
@idoshamun For the article page we could do it, but the modals use slightly different approaches.
And to top it off squads have their own modals.

So for now I think it's better to keep as is, and simply implement on video as well.
With the change to feed refactor this might be something we can adjust there if that works for you?

(Mostly don't want to overcomplicate it here)

Reply:
Yap, that's fine :)